From: Mike Crute <mike@crute.us>
Date: Tue, 24 Dec 2019 16:18:32 +0000
Subject: Disable broken tests

Patch removes check for git info in server version output. This assumes
that the binary was built within a git repo which is not the case for
releases. This test will never work for us.

---

--- a/test/core/test_server.cc
+++ b/test/core/test_server.cc
@@ -51,7 +51,7 @@ TEST_F(ServerTest, ServerOutputsHelpInformation)
     ASSERT_THAT(output.c_str(), HasSubstr(expectedOutput.c_str()));
 }
 
-TEST_F(ServerTest, ServerOutputsCompileInformationIncludingGit)
+TEST_F(ServerTest, DISABLED_ServerOutputsCompileInformationIncludingGit)
 {
     fs::path cmd = fs::path(CMAKE_BINARY_DIR) / "gerbera --compile-info 2>&1";
     std::string output = exec(cmd.c_str());
